README.md

```diff
@@ -1,25 +1,7 @@
-# Rsync - Docker mod for openssh-server
+# kcc - Docker mod for Calibre Web
 
-This mod adds rsync to openssh-server, to be installed/updated during container start.
+This mod adds kcc and its multitude of ebook processing (upscale, stretch, right to left for manga...) to Calibre Web, to be installed/updated during container start.
```

Comment:
Please specify what kcc stands for (Kindle Comic Converter). This is the only description users will see. You want them to know what this mod does.